frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTableManager.java

```diff
@@ -48,6 +54,7 @@
   private final RepositoryGraphFactory repositoryGraphFactory;
   private final VcsRootComboBox vcsRootComboBox;
   private final IGitMacheteRepositoryFactory gitMacheteRepositoryFactory;
+  private final IBranchLayoutParserFactory branchLayoutParserFactory;
 
   public GitMacheteGraphTableManager(Project project, VcsRootComboBox vcsRootComboBox) {
```

Comment:
Comment to line 135: do we want to retain explicit calls to `updateUI()` other than `super.updateUI()`? Maybe forbid them with checkstyle + comment why?

Reply:
resolved! regexp check added